doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
 traffic require the capability to use a user space networking technology.
 
-> Note: An infrastructure can provide the possibility to use SR-IOV with DPDK as
-an additional feature and still be conformant with CNTT.
-
-> Editors note: The possibility to SR-IOV for DPDK is under discussion.
-
-The integration of SDN solutions required by `req.inf.ntw.05` should be enabled
-via their respective CNI integration.
 
-> Note: An SDN solution can manage the pod networks via the Kubernetes API or
-can have a CNI integration.
+| Requirement | Networking Solution with Multus | Networking Solution with DANM |
+|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported |
+| Security Groups (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant |
+| Traffic patterns symmetry (`infra.net.cfg.006`) | Not relevant | Not relevant |
+| Network resiliency (`req.inf.ntw.01`) | Supported | Supported |
+| Centrally administrated and configured (`req.inf.ntw.03`) | Supported via Kubernetes API | Supported via Kubernetes API |
+| Dual stack IPv4 and IPv6 for Kubernetes workloads (`req.inf.ntw.04`) | Supported via the additional CNI plugin | Supported |
+| Integrating SDN controllers (`req.inf.ntw.05`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| More than one networking solution (`req.inf.ntw.06`) | Supported | Supported |
+| Choose whether or not to deploy more than one networking solution (`req.inf.ntw.07`) | Supported | Supported |
+| Kubernetes network model (`req.inf.ntw.08`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| Do not interfere with or cause interference to any interface or network it does not own (`req.inf.ntw.09`) | Supported | Supported |
+| Cluster wide coordination of IP address assignment (`req.inf.ntw.10`) | Supported via the additional CNI plugin | Supported |
 
-The container based architecture must support telecom equipment networking where
-the CNF networks are set up by the operator's network administrators. This is
-why, as `req.gen.cnt.05` requires, the architecture must provide a set of
-abstract management APIs to manage the network connectivity of the CNF pods
-themselves.
+<p align="center"><b>Table 3-1:</b> Comparison of CNI multiplexers/meta-plugins within the networking solution</p>
 
-The API must support multiple tenants and must require elevated access rights to
-manipulate infrastructure related API objects as these operations generally
-require reconfiguration of the physical network infrastructure.
+(1): Under implementation in the current release.  
 
-To fulfill the requirements of `e.cap.016` the architecture should optionally
```

Comment:
I think this should not be removed.

Reply:
Ah good spot - I meant to add this to chapter 4 instead.  Have added in https://github.com/cntt-n/CNTT/commit/21e0cf1f09c7a7ef73daf4a8768b38dd3705549f

Reply:
Is this ok to resolve @CsatariGergely?